### PR TITLE
Feature/gh 607

### PR DESCRIPTION
**Pull request by developpeurweb**
fixes GH-607

### CODE REVIEW

File: projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss [hunk 1]
```diff
@@ -1,189 +1,148 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
-@import '~bootstrap/scss/nav';
-@import '~bootstrap/scss/dropdown';
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss]: There is an unwanted side effect to removing this dropdown import (I think it's the cause) basically you will see on this branch the dropdown has a larger top margin. This import overrides a default 11px from a class `dropdown-menu` in blocks/dropdown. The margin is then 0.125em which imho looks better. You could manually add the top margin to `cx-nav-child-list-columns`. We can talk about it if you want me to show you.

File: projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss [hunk 2]
```diff
@@ -1,189 +1,148 @@
...
+  &-column-title {
+    @include type('7');
+    text-transform: var(--cx-text-transform, capitalize);
+    font-weight: var(--cx-font-weight, var(--cx-g-font-weight-bold));
+    display: var(--cx-display, block);
+    width: var(--cx-width, 100%);
+    padding: var(--cx-padding, 0.25rem 1.5rem);
   }
-  &__child-column {
-    margin-right: $cx-navigation-child-column-trbl-margin;
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss]: Funny story (not an issue) but this rule didn't work before but now it works and it changes the look of the dropdown a bit 😉

File: projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss (hunk 1 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss]: fixed! (without the import)

File: projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss (hunk 2 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss]: fixed!

**Pull request merged**